### PR TITLE
Re-committing because of text errors

### PR DESCRIPTION
**Pull request by owen-adams2021**
image

I'm not finished with the button

### CODE REVIEW

**Comment by owen-adams2021** [comment]
Font* errors, not text errors. Screenshot is in our Slack messages.

**Comment by owen-adams2021** [comment]
image

File: css-two-column-layout/styles.css [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+}
+
+.column-full {
+  text-align: center;
+}
+
+.column-half {
+  display: flex;
+  flex-wrap: flex;
+}
```
Comment (codingcodymiller) [inline css-two-column-layout/styles.css:43]: This doesn't represent a half column as you used it in your document. Each of the elements with this class is a row taking up the full width. I would suggest that you make this a `row` class, then add a `column-half` class (that actually is `50%` width) to each of the `.content` elements.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Nice job so far! A little revision needed:

**Comment by owen-adams2021** [comment]
image

I'm glad you noticed this, at the time I couldn't figure it out so I just swapped paragraphs around.

File: css-two-column-layout/styles.css [hunk 2]
```diff
@@ -28,7 +28,7 @@ p {
   margin-bottom: 2.5rem;
 }
 
-.container {
+.container1 {
```
Comment (codingcodymiller) [inline css-two-column-layout/styles.css:31]: This should just stay `.container`

File: css-two-column-layout/styles.css [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+  margin: 0 auto;
+}
+
+.column-full {
+  text-align: center;
+}
+
+.row {
+  display: flex;
+  flex-wrap: flex;
```
Comment (codingcodymiller) [inline css-two-column-layout/styles.css:42]: This line can be removed, the value is invalid.

File: css-two-column-layout/index.html [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+      <h1>Scale, Speed, Agility</h1>
+    </div>
+    <div class="column-full">
+      <p class="botMargin">We've dedicated thousands of man hours to making healthcare
+        institutions in the Helium<br> network run 10x more efficiently and deliver
+        an unparalleled quality of care.
+      </p>
+    </div>
+    <div class="row">
+      <div class="column-half">
```
Comment (codingcodymiller) [inline css-two-column-layout/index.html:26]: I see you added the `column-half` class, but it doesn't have any styling associated with it. You should create a CSS rule for the class, and apply a `width` property set to `50%`.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Great improvement! A few more changes for you:

File: css-two-column-layout/styles.css (hunk 2 above)
Comment (owen-adams2021) [reply css-two-column-layout/styles.css:31]: image

File: css-two-column-layout/styles.css (hunk 3 above)
Comment (owen-adams2021) [reply css-two-column-layout/styles.css:42]: image

File: css-two-column-layout/index.html (hunk 4 above)
Comment (owen-adams2021) [reply css-two-column-layout/index.html:26]: image

**Comment by owen-adams2021** [comment]
Thank you for clearing this up - flood me with your knowledge.

**Review by codingcodymiller — APPROVED** [review]
Nicely done!!!

**Pull request merged**